packages/storybook-shared/index.d.ts.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,cAAc,YAAY,CAAA"}
```

Comment:
@Jack-Works need update gitignore?

Reply:
No. This should be git tracked because sync it manually from Crowdin is much more complex